src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -72,4 +104,79 @@ export class ExamChecklistComponent implements OnChanges, OnInit, OnDestroy {
         const startedTopic = this.examChecklistService.getStartedTopic(this.exam);
         this.websocketService.unsubscribe(startedTopic);
     }
+
+    /**
+     * Evaluates all the quiz exercises that belong to the exam
+     */
+    evaluateQuizExercises() {
+        this.isEvaluatingQuizExercises = true;
+        if (this.exam.course?.id !== undefined && this.exam.id !== undefined) {
+            this.examManagementService.evaluateQuizExercises(this.exam.course.id, this.exam.id).subscribe({
+                next: (res) => {
+                    this.alertService.success('artemisApp.studentExams.evaluateQuizExerciseSuccess', { number: res?.body });
+                    this.existsUnassessedQuizzes = false;
+                    this.isEvaluatingQuizExercises = false;
+                },
+                error: (err: HttpErrorResponse) => {
+                    this.handleError('artemisApp.studentExams.evaluateQuizExerciseFailure', err);
+                    this.isEvaluatingQuizExercises = false;
+                },
+            });
+        } else {
+            throw new Error(`Cannot evaluate quiz exercises due to missing course or exam id.`);
```

Comment:
this should probably be done with alertService.error + a translate string (also for the method below)

Reply:
Changed